### PR TITLE
Implement Minigames base into game and adding localization.

### PR DESCRIPTION
**Pull request by rster2002**
## Changes
Creates a base for how minigames will be implemented and adds the first minigame into the game.

## Notes
Keep track of progress in this draft PR.

### CODE REVIEW

File: Demo/Demo/Assets/Scripts/Controllers/MinigameController.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        _minigames.Add(password);
+        _minigames.ForEach(x => x.SubscribeToCompletion(OnMinigameCompletion));
+    }
+
+    public void Activate(MinigameInteraction interaction, MinigameType type)
+    {
+        _currentInteraction = interaction;
+        _currentInteraction.OnExit += OnMiniGameExit;
+
+        _minigames.ForEach(x =>
```
Comment (rster2002) [inline Demo/Demo/Assets/Scripts/Controllers/MinigameController.cs:32]: Replace `x` with `minigame`

File: Demo/Demo/Assets/Scripts/MiniGame001.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            txtInput.enabled = false;
+            txtWrongPassword.color = Color.red;
+            RestartGame();
+        }
+    }
+
+    private string CreateRandomPassword()
+    {
+        int min = 0;
+        int max = 10;
```
Comment (rster2002) [inline Demo/Demo/Assets/Scripts/MiniGame001.cs:72]: The max should be 4

**Review by rster2002 — COMMENTED** [review]
Number-pad is non-standard layout. This would be more in line with your phone for example:

images

**Comment by rster2002** [comment]
Appliances didn't break down during my testing.

File: Demo/Demo/Assets/Scripts/Controllers/MinigameController.cs (hunk 1 above)
Comment (shukerb) [reply Demo/Demo/Assets/Scripts/Controllers/MinigameController.cs]: Done

File: Demo/Demo/Assets/Scripts/MiniGame001.cs (hunk 2 above)
Comment (shukerb) [reply Demo/Demo/Assets/Scripts/MiniGame001.cs]: removed the min and max vars

**Review by suchmememanyskill — APPROVED** [review]

**Comment by rster2002** [comment]
Couldn't build on android, please check if you can fix it.

**Pull request merged**